src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkUtils.java

```diff
@@ -130,27 +147,42 @@ private static void createBucket(Storage storage, String bucket, @Nullable Strin
     }
   }
 
+  /**
+   * Updates {@link Configuration} with bucket details.
+   * Uses provided bucket, otherwise generates random.
```

Comment:
It's actually not plain random, it's equal to runid.

Reply:
Good point. let me fix this.